src/components/grapher/Grapher.vue

```diff
@@ -285,4 +308,9 @@ export default Vue.extend({
     opacity: 0.5;
   }
 }
+.grapher--dottext {
```

Comment:
nit: Can you also add `user-select: none` so that the mouse icon doesn't turn into a text icon? https://developer.mozilla.org/en-US/docs/Web/CSS/user-select

Reply:
Yep!

Reply:
Fixed in latest PR.